### PR TITLE
128.0

### PR DESCRIPTION
**Pull request by coreycb**

### CODE REVIEW

File: release/128.0.yml [hunk 1]
```diff
@@ -0,0 +1,796 @@
...
+    - 1GB RAM / 2GB of RAM for the 64-bit version
+    - 250 MB hard drive space
+    - additional hard drive space for mail storage
+
+    ---
+
+    ## Mac
+    - macOS 10.15 or later
+
+    ### MAC Recommended Hardware
```
Comment (JohnVeness) [inline release/128.0.yml:28]:
```suggestion
    ### Mac Recommended Hardware
```

File: release/128.0.yml [hunk 2]
```diff
@@ -0,0 +1,796 @@
...
+    tag: new
+    bugs: [1837152]
+  - note: OpenPGP no longer attempts to re-initialize secret key storage if individual
+      OpenPGP secret key passphrases have been enabled
+    tag: new
+    bugs: [1842629]
+  - note: OpenPGP Key Manager can now create revocation statements for any secret key
+    tag: new
+    bugs: [1852565]
+  - note: A nested OpenPGP messages' signature status is now viewable
```
Comment (JohnVeness) [inline release/128.0.yml:100]:
```suggestion
  - note: A nested OpenPGP message's signature status is now viewable
```

File: release/128.0.yml [hunk 3]
```diff
@@ -0,0 +1,796 @@
...
+    tag: new
+    bugs: [1861703]
+  - note: Changing expiration of OpenPGP keys with a "complex" structure now supported
```
Comment (JohnVeness) [inline release/128.0.yml:103]:
```suggestion
  - note: Changing expiration of OpenPGP keys with a "complex" structure is now supported
```

File: release/128.0.yml [hunk 4]
```diff
@@ -0,0 +1,796 @@
...
+  - note: Updated donation link from "give.thunderbird.net" to "www.thunderbird.net/donate/"
+    tag: changed
+    bugs: [1820676]
+  - note: Activity Manager now shows account and full folder path when indexing
+    tag: changed
+    bugs: [1823024]
+  - note: Updated message pane customization defaults for new profiles
+    tag: changed
+    bugs: [1840891]
+  - note: Cards UI styling of expand and collapse for threads
```
Comment (JohnVeness) [inline release/128.0.yml:154]: Not really a sentence. How did it change?

File: release/128.0.yml [hunk 5]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1893899]
+  - note: Clicking on native Windows notifications did not bring Thunderbird to the foreground
+    tag: fixed
+    bugs: [1863798]
+  - note: Commandline "--migration" option did not work
+    tag: fixed
+    bugs: [1901912]
+  - note: All Thunderbird notifications on Windows are now removed when Thunderbird is
+      closed
```
Comment (JohnVeness) [inline release/128.0.yml:248]:
```suggestion
  - note: Thunderbird notifications on Windows were not always removed when Thunderbird was
      closed
```

File: release/128.0.yml [hunk 6]
```diff
@@ -0,0 +1,796 @@
...
+  - note: Windows taskbar icon did not have the "Beta" label
+    tag: fixed
+    bugs: [1866730]
+  - note: Some menu bar items did not work with all Thunderbird windows closed on macOS
+    tag: fixed
+    bugs: [1868718]
+  - note: Various visual and UX improvements
+    tag: fixed
+    bugs: [1855611, 1869294, 1870323, 1870331, 1870705, 1871001, 1872873, 1873236]
+  - note: Sender in multimesage view was right aligned when the text wrapped to multiple
```
Comment (JohnVeness) [inline release/128.0.yml:270]:
```suggestion
  - note: Sender in multi-message view was right aligned when the text wrapped to multiple
```

File: release/128.0.yml [hunk 7]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1881100]
+  - note: '"Find in this message" did not work in multi-message view'
+    tag: fixed
+    bugs: [1839005]
+  - note: 'Non-compliant "References" headers were not handled gracefully when `mailnews.headers.showReferences`
+      was set'
+    tag: fixed
+    bugs: [1888116]
+  - note: '"news:// URIs no longer opened as links or from the commandline'
```
Comment (JohnVeness) [inline release/128.0.yml:284]:
```suggestion
  - note: '"news:// URIs failed opening as links or from the commandline'
```

File: release/128.0.yml [hunk 8]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1889971]
+  - note: Focus did not transfer to a mail message opened in a new tab
+    tag: fixed
+    bugs: [1886653]
+  - note: Focus behavior for opening messages in a new tab changed depending on state of
+      "mail.tabs.loadInBackground" preference
+    tag: fixed
+    bugs: [1894831]
+  - note: Focus changes in 3-pane occassionally caused keyboard shortcuts to stop working
```
Comment (JohnVeness) [inline release/128.0.yml:334]:
```suggestion
  - note: Focus changes in 3-pane view occassionally caused keyboard shortcuts to stop working
```

File: release/128.0.yml [hunk 9]
```diff
@@ -0,0 +1,796 @@
...
+  - note: Incorrect message was selected under certain circumstances
+    tag: fixed
+    bugs: [1846885]
+  - note: Broken folders were excluded from folder tree
+    tag: fixed
+    bugs: [1853824]
+  - note: Cards View UX improvements
+    tag: fixed
+    bugs: [1855821, 1855823, 1856623, 1857613, 1857627, 1859036]
+  - note: Star and junk icons showed incorrect state for selected-focused messages in table
```
Comment (JohnVeness) [inline release/128.0.yml:353]:
```suggestion
  - note: Star and Junk icons showed incorrect state for selected-focused messages in table
```

File: release/128.0.yml [hunk 10]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1855643]
+  - note: Various Cards View improvements
+    tag: fixed
+    bugs: [1882592, 1890387, 1890388, 1890674]
+  - note: Various style and UI fixes
+    tag: fixed
+    bugs: [1845204, 1849315, 1865615, 1888750, 1888812, 1889021, 1889519, 1890044, 1890129,
+      1890345]
+  - note: Thread state in Cards view was not updated after toggling Group By sort order.
```
Comment (JohnVeness) [inline release/128.0.yml:417]:
```suggestion
  - note: Thread state in Cards View was not updated after toggling Group By sort order
```

File: release/128.0.yml [hunk 11]
```diff
@@ -0,0 +1,796 @@
...
+    bugs: [1901926]
+  - note: Scrolling to an unread message in an expanded thread did not show as much of
+      the thread as possible
+    tag: fixed
+    bugs: [1900916]
+  - note: In a synthetic view, right-clicking an unselected message incorrectly selected
+      the message
+    tag: fixed
+    bugs: [1902549]
+  - note: In cards view, when all child messages of a thread were read, the replies button
```
Comment (JohnVeness) [inline release/128.0.yml:466]:
```suggestion
  - note: In Cards View, when all child messages of a thread were read, the replies button
```

File: release/128.0.yml [hunk 12]
```diff
@@ -0,0 +1,796 @@
...
+      option is now available in List-Id menu'
+    tag: new
+    bugs: [512385]
+  - note: Threaded message views now include "New Message" count
+    tag: new
+    bugs: [1883506]
+  - note: Improved Cards View UI
+    tag: new
+    bugs: [1877324, 1882369]
+  - note: '"Cards View" now shows a "new" icon when there is a new message in a thread'
```
Comment (JohnVeness) [inline release/128.0.yml:83]:
```suggestion
  - note: 'Cards View now shows a "new" icon when there is a new message in a thread'
```

File: release/128.0.yml [hunk 13]
```diff
@@ -0,0 +1,796 @@
...
+  - note: Updated message pane customization defaults for new profiles
+    tag: changed
+    bugs: [1840891]
+  - note: Cards UI styling of expand and collapse for threads
+    tag: changed
+    bugs: [1860868]
+  - note: Spam and Junk message styling in Card UI adjusted to be less distracting
+    tag: changed
+    bugs: [1880163]
+  - note: '"Cards View" Grouped By Sort style improvements'
```
Comment (JohnVeness) [inline release/128.0.yml:160]:
```suggestion
  - note: 'Cards View Grouped By Sort style improvements'
```

File: release/128.0.yml [hunk 14]
```diff
@@ -0,0 +1,796 @@
...
+      unselected folder
+    tag: fixed
+    bugs: [1901386]
+  - note: View menu in message window did not initialize correctly
+    tag: fixed
+    bugs: [1862665]
+  - note: Message List display options were hidden outside of Thunderbird window
+    tag: fixed
+    bugs: [1854393]
+  - note: Visual and UX Quick Filter improvements
```
Comment (JohnVeness) [inline release/128.0.yml:490]:
```suggestion
  - note: Quick Filter visual and UX improvements
```

File: release/128.0.yml [hunk 15]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1866717, 1892436, 1892560]
+  - note: Resetting the global search bar content loses focus
```
Comment (JohnVeness) [inline release/128.0.yml:493]:
```suggestion
  - note: Resetting the global search bar content lost focus
```

File: release/128.0.yml [hunk 16]
```diff
@@ -0,0 +1,796 @@
...
+    bugs: [1893039]
+  - note: S/MIME status (encryption and signature) was not shown when opening an email
+      from a file
+    tag: fixed
+    bugs: [1902991]
+  - note: Add tooltip displaying full folder path when hovering over location column of
+      thread pane
+    tag: fixed
+    bugs: [522768]
+  - note: Clear button was missing from Unified toolbar customization search bar
```
Comment (JohnVeness) [inline release/128.0.yml:533]:
```suggestion
  - note: Clear button was missing from Unified Toolbar customization search bar
```

File: release/128.0.yml [hunk 17]
```diff
@@ -0,0 +1,796 @@
...
+  - note: References and Reply-To headers were not set if they were included in `mail.compose.other.header`
+    tag: fixed
+    bugs: [1800759]
+  - note: Message compose window was not fully disabled while message was sending
+    tag: fixed
+    bugs: [1758469]
+  - note: Delivery Status Notification did not work when sending message using "Send Later"
+    tag: fixed
+    bugs: [815638]
+  - note: Print dialog opened in Compose window did not prevent window interaction
```
Comment (JohnVeness) [inline release/128.0.yml:549]:
```suggestion
  - note: Print dialog opened in Message compose window did not prevent window interaction
```

File: release/128.0.yml [hunk 18]
```diff
@@ -0,0 +1,796 @@
...
+    bugs: [1884661]
+  - note: Messages moved to junk folder from message context menu were marked as read,
+      regardless of whether "Mark messages determined to be junk as read" was checked/unchecked
+      in settings
+    tag: fixed
+    bugs: [1860444]
+  - note: New posts retrieved from RSS feeds incorrectly played mail alert sound
+    tag: fixed
+    bugs: [1871493]
+  - note: '"Close" button in Cookie Manager is now more discoverable on macOS'
```
Comment (JohnVeness) [inline release/128.0.yml:586]:
```suggestion
  - note: 'No "Close" button in Cookie Manager on macOS'
```

File: release/128.0.yml [hunk 19]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1872490]
+  - note: Chat Account Settings did not allow scrolling to view overflowed window contents
+    tag: fixed
+    bugs: [1868796]
+  - note: Account setup triggered a second OAuth when looking for calendars
+    tag: fixed
+    bugs: [1880646]
+  - note: Google Calendar account detection failed for Google hosted domains when the DNS
+      entry had Upper-cased characters
```
Comment (JohnVeness) [inline release/128.0.yml:604]:
```suggestion
  - note: Google Calendar account detection failed for Google-hosted domains when the DNS
      entry had uppercase characters
```

File: release/128.0.yml [hunk 20]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1841336]
+  - note: LDAP addressbook did not work
+    tag: fixed
+    bugs: [1841348]
+  - note: macOS Contacts integration with Thunderbird address book did not work
+    tag: fixed
+    bugs: [1864521]
+  - note: Adding duplicate mail addresses in "New/Edit mailing list" dialog, pressing OK,
+      then removing one of the duplicate addresses causes both addresses to be removed
```
Comment (JohnVeness) [inline release/128.0.yml:638]:
```suggestion
      then removing one of the duplicate addresses caused both addresses to be removed
```

File: release/128.0.yml [hunk 21]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1870873]
+  - note: CardDAV addressbooks could not be used as an import target
+    tag: fixed
+    bugs: [1893876]
+  - note: Currently selected address book icon was solid color when creating a new contact
+    tag: fixed
+    bugs: [1901547]
+  - note: Removing Organizational Properties from a contact did not clear all organization
+      related fields
```
Comment (JohnVeness) [inline release/128.0.yml:651]:
```suggestion
  - note: Removing Organizational Properties from a contact did not clear all
      organization-related fields
```

File: release/128.0.yml [hunk 22]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1901739]
+  - note: CardDAV contacts synced to server when no changes were made
```
Comment (JohnVeness) [inline release/128.0.yml:654]:
```suggestion
  - note: CardDAV contacts were synced to server when no changes were made
```

File: release/128.0.yml [hunk 23]
```diff
@@ -0,0 +1,796 @@
...
+  - note: Various UX and visual improvements
+    tag: fixed
+    bugs: [1852733]
+  - note: High Contrast theme improvements
+    tag: fixed
+    bugs: [1890682]
+  - note: Compose window toolbar items were unreadable on macOS in dark mode
+    tag: fixed
+    bugs: [1893249]
+  - note: Dark mode colors were inconsistent throughout application
```
Comment (JohnVeness) [inline release/128.0.yml:676]:
```suggestion
  - note: Dark mode colors were inconsistent throughout Thunderbird
```

File: release/128.0.yml [hunk 24]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1862208]
+  - note: Remote content warning was shown multiple times for the same message
+    tag: fixed
+    bugs: [1875536]
+  - note: IMAP accounts with missing or incorrect OAuth credentials sometimes caused duplicate
+      dialogs to appear, requesting a new OAuth token
+    tag: fixed
+    bugs: [1880211]
+  - note: Hardware key multi-factor authentication did not in Thunderbird Flatpak
```
Comment (JohnVeness) [inline release/128.0.yml:741]:
```suggestion
  - note: Hardware key multi-factor authentication did not work in Thunderbird Flatpak
```

File: release/128.0.yml [hunk 25]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1882985]
+  - note: GPG smartcards did not work in Thunderbird for macOS
```
Comment (JohnVeness) [inline release/128.0.yml:744]:
```suggestion
  - note: GPG smartcards did not work in macOS
```

File: release/128.0.yml [hunk 26]
```diff
@@ -0,0 +1,796 @@
...
+    tag: fixed
+    bugs: [1890975]
+  - note: Calendar 'Find Events Pane' did not match search terms if they included uppercase
+      letters
+    tag: fixed
+    bugs: [1898856]
+  - note: Calendar startup performance improvements
+    tag: fixed
+    bugs: [1856509]
+  - note: Dropmarker in Today Pane was not centered
```
Comment (JohnVeness) [inline release/128.0.yml:767]:
```suggestion
  - note: Drop-down icon in Today Pane was misaligned
```

File: release/128.0.yml [hunk 27]
```diff
@@ -0,0 +1,796 @@
...
+  - note: Task categories menu did not display category colors
+    tag: fixed
+    bugs: [1892673]
+  - note: Keyboard accessibility fixes
+    tag: fixed
+    bugs: [1898199]
+  - note: 'Context menu with "Copy link" option was not available in calendar event descriptions'
+    tag: fixed
+    bugs: [1804643]
+  - note: '"Item changed on server" Calendar dialog did not have a scrollbar for lengthy
```
Comment (JohnVeness) [inline release/128.0.yml:783]:
```suggestion
  - note: '"Item changed on server" calendar dialog did not have a scrollbar for long
```

**Comment by coreycb** [comment]
@JohnVeness I've made updates based on your comments. Thanks for the review!

File: release/128.0.yml [hunk 28]
```diff
@@ -0,0 +1,525 @@
...
+    different requirements.*
+
+    - Thunderbird will not run at all without the following libraries or packages:
+        - glibc 2.17 or higher
+        - GTK+ 3.14 or higher
+        - libdbus-glib 0.6.0 or higher
+        - libglib 2.42 or higher
+        - libstdc++ 4.8.1 or higher
+        - libxtst 1.2.3 or higher
+        - X.Org 1.0 or higher (1.7 or higher is recommended)
```
Comment (jfx2006) [inline release/128.0.yml:49]: These changed in 128.0, the 128.0beta.yml file has the updated Linux requirements.

File: release/128.0.yml [hunk 29]
```diff
@@ -0,0 +1,525 @@
...
+    bugs: [512385]
+  - note: Threaded message views now include "New Message" count
+    tag: new
+    bugs: [1883506]
+  - note: Cards View hover and selection paradigms introduced to better support custom
+      themes and accent colors
+    tag: new
+    bugs: [1857120, 1863614]
+  - note: Improved Cards View readability and accessibility
+    tag: new
```
Comment (jfx2006) [inline release/128.0.yml:76]: Cards View was new for 115, so "changed"?

File: release/128.0.yml [hunk 30]
```diff
@@ -0,0 +1,525 @@
...
+    tag: fixed
+    bugs: [1868718]
+  - note: Sender in multi-message view was right aligned when the text wrapped to multiple
+      lines
+    tag: fixed
+    bugs: [1880252]
+  - note: '"Find in this message" did not work in multi-message view'
+    tag: fixed
+    bugs: [1839005]
+  - note: '"news:// URIs failed opening as links or from the commandline'
```
Comment (jfx2006) [inline release/128.0.yml:216]: "news://"

File: release/128.0.yml [hunk 31]
```diff
@@ -0,0 +1,590 @@
...
+    **Thunderbird version 128.0 is only offered as direct download from
+    thunderbird.net and not as an upgrade from Thunderbird version 115 or
+    earlier. A future release will provide updates from earlier versions.**
+
+    **For more on all the new features in Thunderbird 128, see
+    [What’s New in Thunderbird 128](https://www.thunderbird.net/thunderbird/128.0/whatsnew/).**
+
+    **System Requirements:** [Details](/en-US/thunderbird/128.0/system-requirements/)
+
+    system_requirements: |
```
Comment (MelissaAutumn) [inline release/128.0.yml:13]: Too much whitespace, it's lumped in with the text.
image

File: release/128.0.yml [hunk 32]
```diff
@@ -0,0 +1,590 @@
...
+        - libglib 2.42 or higher
+        - libstdc++ 4.8.1 or higher
+        - X.Org 1.0 or higher (1.7 or higher is recommended)
+
+    - For optimal functionality, we recommend the following libraries or packages:
+        - DBus 1.0 or higher
+        - NetworkManager 0.7 or higher
+        - PulseAudio
+
+  import_system_requirements: '128.0'
```
Comment (MelissaAutumn) [inline release/128.0.yml:54]: You can remove this as you define system requirements here.

**Review by MelissaAutumn — CHANGES_REQUESTED** [review]
Some things I caught while testing the 128.0esr question

File: release/128.0.yml (hunk 32 above)
Comment (coreycb) [reply release/128.0.yml:54]: Oops, I thought I fixed that. Thank you!

**Review by MelissaAutumn — APPROVED** [review]
My requested changes are fixed. So approving this in that context, not the entire PR. 😄

**Pull request merged**